examples/3-mutation/src/Grid.re

```diff
@@ -30,34 +18,26 @@ module GetAllDogs = [%graphql
 |}
 ];
 
-let request = GetAllDogs.make();
-
 [@react.component]
 let make = () => {
-  let (Hooks.{response}, _) = Hooks.useQuery(~request, ());
+  let ({Hooks.data}, _) = UseQuery.useQuery(~query=(module GetAllDogs), ());
```

Comment:
I'd really prefer us to use the `response` variant in all examples and always reference hooks from the top-level `Hooks` module. The main reason is to push users towards comprehensively handling error, fetching, and partial data states.

Reply:
In the new ReScript (8.2+) no annotation is necessary here for the record.